index.js

```diff
@@ -29,33 +29,36 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-'use strict';
+"use strict";
```

Comment:
Our other APIs always use single quotes-- doesn't really matter to me, but was the change to `"`s intentional, or maybe the auto-linter ran off and changed them on its own?